packages/api-plugin-fulfillment/migrations/2.js

```diff
@@ -0,0 +1,47 @@
+const COLL_FF_SOURCE = "Shipping";
+const COLL_FF_DEST = "Fulfillment";
+const COLL_FFR_SOURCE = "FlatRateFulfillmentRestrictions";
+const COLL_FFR_DEST = "FulfillmentRestrictions";
+
```

Comment:
What is the reasoning behind breaking this into three different migrations? This is definitely not the pattern we should follow.

Reply:
As discussed, I will be combining all the migrations into one. This will be implemented in the new replacement PR

Reply:
I have implemented here also and will be copied over to the new replacement PR